### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#340] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325171-1e6a4261-c383-4832-b998-89ca2faab9c4.png)

**Comment by LuginiAndrea** [comment]
1. $$a_1 \lor a_2 \lor a_3 \lor a_4$$
2. $$\bigvee_{i=1}^4 a_i \land \bigvee_{i=1}^4 b_i \land \bigwedge_{i=1}^4 \neg\left(a_i \land b_i\right)$$ Spiegazione: le prime due clausole ci dicono che i due sottoinsiemi non sono vuoti. L'ultima ci assicura che se un valore è presente sia in $A$ che in $B$ allora la proposizione è falsa
3. $$\bigvee_{i=1}^4 a_i \land \bigvee_{i=1}^4 b_i \land \bigwedge_{i=1}^4 a_i \lor b_i$$
4.  $$\bigvee_{i=1}^4 a_i \land \bigvee_{i=1}^4 b_i \land \bigwedge_{i=1}^4 \left(a_i \land b_i \lor \neg a_i\right) \land \bigvee_{i=1}^4 \left(\neg a_i\right) \land b_i$$ Spiegazione: la terza clausola ci dice che un elemento o esiste sia in $A$ che in $B$ oppure non esiste in $A$ (Praticamente, tutti gli elementi che esistono in $A$ esistono anche in $B$). La quarta clausola ci dice che almeno un elemento che esiste in $B$ non esiste in $A$, il che ci dice che $B$ è diverso da $A$
